public/index.html

```diff
@@ -202,13 +203,16 @@ <h1>Congratulations!</h1>
               <button onclick="reloadPage()">Play again</button>
             </p>
             <p>
-              <a id="share-via-twitter" class="button smaller"
-                >Share with Twitter</a
-              >
-              <a id="share-via-facebook" class="button smaller"
-                >Share with Facebook</a
-              >
+              <a id="share-results" class="button smaller">Share</a>
             </p>
+            <p></p>
```

Comment:
That empty paragraph is probably not needed?

Reply:
That was some weirdness that prettier added, but yeah, I think we can take it out

Reply:
Oh, I remember now, I was trying to put that next div into a <p> block to make sure it was separated from the share button when it's visible, and prettier refused to have that for some reason. But I think we're OK because there's enough padding that it looks OK even with the empty paragraph gone.